integration-cli/docker_utils.go

```diff
@@ -269,6 +269,14 @@ func (d *Daemon) Cmd(name string, arg ...string) (string, error) {
 	return string(b), err
 }
 
+func (d *Daemon) CmdWithArgs(daemonArgs []string, name string, arg ...string) (string, error) {
```

Comment:
I don't really like the idea of something almost identical to `d.Cmd` can we not refactor the other to work for both?